content/sandbox.markdown

```diff
@@ -10,20 +10,36 @@ excerpt: Welcome to DNSimple. This page explains our sandbox environment. Enjoy
 
 ---
 
-We maintain an isolated test site at [sandbox.dnsimple.com](https://sandbox.dnsimple.com/). Please feel free to test your applications that integrate with the DNSimple API here.
+We maintain an isolated test site at [sandbox.dnsimple.com](https://sandbox.dnsimple.com/). Please feel invited to test your applications that integrate with the DNSimple API here.
+
+### Disclaimer
 
 Sandbox data and systems are generally treated like production, however we may not apply the same backup, monitoring and high availability policies. That means we may occasionally need to take this environment down, wipe out a portion, or clear all stored data.
 
+## Getting started
+
+The DNSimple sandbox is a independent system, you have to create another account and register some domains to test your integrations.
```

Comment:
It may be worth mentioning that the account and domains will be free of charge so it doesn't scare anyone off at this point of the read.